src/chunk.c

```diff
@@ -1410,3 +1410,46 @@ chunk_set_schema(Chunk *chunk, const char *newschema)
 
 	return chunk_update_form(&chunk->fd);
 }
+
+/* Used as a tuple found function */
+static bool
+chunk_rename_schema_name(TupleInfo *ti, void *data)
+{
+	HeapTuple	tuple = heap_copytuple(ti->tuple);
+	FormData_chunk *chunk = (FormData_chunk *) GETSTRUCT(tuple);
+
+	/* Rename schema name */
+	namestrcpy(&chunk->schema_name, (char *) data);
+	catalog_update(ti->scanrel, tuple);
+	heap_freetuple(tuple);
+	/* Return true to keep going */
+	return true;
+}
+
+/* Go through the internal chunk table and rename all matching schemas */
+void
+chunks_rename_schema(char *old_schema, char *new_schema)
```

Comment:
similar to above, this func is mostly the same as `dimension_rename_schema_name`, can it be refactored to just take in the args that change?

(if not, at least this function should mirror the naming and be `chunk_rename_schema_name`)

Reply:
The name change doesn't work since this is a function that goes through all chunks in the table (hence the "s") and makes a change on each chunk that should be renamed. So the function that renames a single chunk has the name you described.

I'm open to other naming options if you don't like `chunks_rename_schema`.

Reply:
`chunks_rename_schema_name` is probably more symmetrical.

Reply:
fixed